doc/01_Getting_Started/05_Handle_Failed_Jobs.md

```diff
@@ -0,0 +1,18 @@
+# Handle Failed Jobs
+
+If some maintenance jobs are failed in the processing, they are discarded from the respective transport after defined retries. 

```

Comment:
```suggestion
If maintenance jobs fail during processing, they are discarded from the respective transport after defined number of retries.
```

Reply:
Nice thanks! We restructured the files a bit, but we added your suggestions!